config/enrichments/16_standardize_mac.conf

```diff
@@ -65,6 +65,14 @@ filter {
         ]
       }
     }
+    if [related.mac] {
```

Comment:
this is added only in enrichment 17_related_mac so it doesn't exist here and is not required. Please remove